src/components/KeyboardShortcutsModal.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import {View} from 'react-native';
+import _ from 'underscore';
+import HeaderWithCloseButton from './HeaderWithCloseButton';
+import Text from './Text';
+import Modal from './Modal';
+import CONST from '../CONST';
+import styles from '../styles/styles';
+import withWindowDimensions, {windowDimensionsPropTypes} from './withWindowDimensions';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import compose from '../libs/compose';
+import KeyboardShortcut from '../libs/KeyboardShortcut';
+
+const propTypes = {
+    /** prop to fetch screen width */
+    ...windowDimensionsPropTypes,
+
+    /** props to fetch translation functions */
+    ...withLocalizePropTypes,
+};
+
+class KeyboardShortcutsModal extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            isOpen: false,
+        };
+
+        this.showKeyboardShortcutModal = this.showKeyboardShortcutModal.bind(this);
+        this.hideKeyboardShortcutModal = this.hideKeyboardShortcutModal.bind(this);
+    }
+
+    componentDidMount() {
+        const shortcutConfig = CONST.KEYBOARD_SHORTCUTS.SHORTCUT_MODAL;
+        const shortcutModifiers = KeyboardShortcut.getShortcutModifiers(shortcutConfig.modifiers);
+        this.unsubscribeShortcutModal = KeyboardShortcut.subscribe(shortcutConfig.shortcutKey, () => {
+            this.showKeyboardShortcutModal();
+        }, shortcutConfig.descriptionKey, shortcutModifiers, true);
+    }
+
+    componentWillUnmount() {
+        if (!this.unsubscribeShortcutModal) {
+            return;
+        }
+        this.unsubscribeShortcutModal();
+    }
+
+    showKeyboardShortcutModal() {
+        this.setState({isOpen: true});
+    }
+
+    hideKeyboardShortcutModal() {
+        this.setState({isOpen: false});
+    }
+
+    renderRow(shortcut, isFirstRow) {
```

Comment:
Sorry if I missed this in previous reviews, but we should really have a JSDoc to describe these parameters.